### ISSUE TITLE
Add a user account update form

### ISSUE THREAD

**Issue by OrdinaryJellyfish** [netlify/netlify-identity-widget#105]
It would be really nice if there was a form where users could update their account details like name, e-mail, and password, because there's an e-mail change template in the Identity settings, but there's no form to do it.

**Comment by QuantumInformation** [comment]
Yeah this would be awesome, saves having to basically create the form from scratch because every site will need them.

**Comment by QuantumInformation** [comment]
I decided to avoid the widget and attempt to make a wrapper around gotrue for svelte.

https://github.com/QuantumInformation/svelte-netlify-identity
